package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "airswap.js",
-  "version": "0.0.28",
```

Comment:
version 0.0.28 is good actually, latest on the registry is 0.0.27

https://www.npmjs.com/package/airswap.js